tests/settings/test_settings_localized_attributes_meilisearch.py

```diff
@@ -0,0 +1,35 @@
+NEW_LOCALIZED_ATTRIBUTES = [{"attributePatterns": ["title"], "locales": ["eng"]}]
+
+
+def test_get_localized_attributes(empty_index):
+    """Tests getting default localized_attributes."""
+    response = empty_index().get_localized_attributes()
+    assert response is None
+
+
+def test_update_localized_attributes(empty_index):
+    """Tests updating proximity precision."""
+    index = empty_index()
+    response = index.update_localized_attributes(NEW_LOCALIZED_ATTRIBUTES)
+    update = index.wait_for_task(response.task_uid)
+    assert update.status == "succeeded"
+    response = index.get_localized_attributes()
+    assert NEW_LOCALIZED_ATTRIBUTES == response
```

Comment:
I believe this may fail after updating to a class, if it does changing to `assert [x.model_dump for x in NEW_LOCALIZED_ATTRIBUTES] == [x.model_dump() for x in response]` should fix it.